### DISCUSSION TITLE
Global modification of services

### DISCUSSION THREAD

**Discussion by PalAditya** [Kong/kong#10262] [Help] [answered]
Is there a way to globally disable retries for all the services in a declarative config file for Kong? I know it can be done per service with "retries: 0", but can we set the default to 0 rather than 5?

**Comment by locao** [comment, marked as answer]
No, if the `retries` value is empty, it will be set as 5: https://github.com/Kong/kong/blob/3.1.1/kong/db/schema/entities/services.lua#L32
